db/migrations/20230801113000_add-capability-status.sql

```diff
@@ -0,0 +1,22 @@
+-- 2023-08-01 11:30:xx : Add status and modified_at to capabilities
+
+-- add columns
+ALTER TABLE "Capability"
+ADD column "Status" varchar(255) NOT NULL DEFAULT 'Active',
+ADD column "ModifiedAt" timestamp NOT NULL DEFAULT clock_timestamp();
```

Comment:
Just curious: are these timestamps in UTC time?

Reply:
They are not, but that is not truly relevant here.
We must have a default value, but it is never used due to the rest of the code.
Added comment around this.